custom/enikshay/nikshay_datamigration/factory.py

```diff
@@ -129,6 +128,12 @@ def get_person_case_structure(self):
             },
         }
 
+        if self._location:
```

Comment:
I think we should check that this location has location_type "phi", otherwise it should fail.